packages/ida.plugin.dereferencing.vm/tools/chocolateyinstall.ps1

```diff
@@ -4,5 +4,6 @@ Import-Module vm.common -Force -DisableNameChecking
 $pluginName = 'dereferencing.py'
 $pluginUrl = 'https://github.com/danigargu/deREferencing/archive/ab106c1fa5969a5d7c27c8037b4aec446ad55a4d.zip'
 $pluginSha256 = '83c0f9420075dbe5217f5fbf2413c7b1b72eadfaedfe2474b306e1e3eaae0aa8'
+$pluginFiles = @('dereferencing', 'dereferencing.py')
```

Comment:
I think this is a bad example to use the `$pluginFiles` argument as this plugin is installed correctly with the default behavior and this change unnecessarily complicates the code. What about removing this change and run `scripts/utils/create_package_template.py` to create the IDA plugin `ida.plugin.hexrayspytools.vm` so that we can test the new argument with an IDA plugin that needs it?

Reply:
does dereferencing fails to uninstall with the current implementation? :thinking:

Reply:
Well technically it doesn't fail, but it doesn't properly uninstall, since it will only remove `dereferencing.py`: https://github.com/mandiant/VM-Packages/blob/7f5911d03424d7da3e47b28915be5c77a4d715fa/packages/ida.plugin.dereferencing.vm/tools/chocolateyuninstall.ps1#L4

The `dereferencing` module folder will remain: https://github.com/danigargu/deREferencing/tree/259685bce8ab4ba0371ec6d13446cc553df2689b/dereferencing

Trying to reinstall the deferencing plugin will then fail because that folder already exists.